DATA_LAYER_INTEGRATION.md

```diff
@@ -0,0 +1,280 @@
+<!--
+Copyright 2020 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Data Layer Integration with the Core Components
+
+The Core Components provide an out-of-the-box integration with the [Adobe Client Data Layer](https://github.com/adobe/adobe-client-data-layer), which for convenience is called data layer in this page.
+
+## Enabling the Data Layer
+
+The data layer is disabled by default. 
+
+To enable the data layer for your site:
+1. Create the following structure below the `/conf` node:
+    `/conf/<my-site>/sling:configs/com.adobe.cq.wcm.core.components.internal.DataLayerConfig`
+1. Add the `enabled` boolean property and set it to `true`.
```

Comment:
There's an extra step needed:
- Add a `sling:configRef` to the root page `jcr:content` poiting the `/conf/<my-site>`

Reply:
Addressed in https://github.com/adobe/aem-core-wcm-components/pull/1026/commits/3aef3ebdf90b8c4c43161e1cb2ebfca204c5edd1